crypto/blake2b.c

```diff
@@ -186,6 +187,30 @@ int blake2b_InitKey( blake2b_state *S, size_t outlen, const void *key, size_t ke
   return 0;
 }
 
+#if OPTIMIZE_SIZE
+static void G(uint64_t *m, int r, int i, uint64_t *a, uint64_t *b, uint64_t *c,
+               uint64_t *d) {
+  *a = *a + *b + m[blake2b_sigma[r][2 * i + 0]];
+  *d = rotr64(*d ^ *a, 32);
+  *c = *c + *d;
+  *b = rotr64(*b ^ *c, 24);
+  *a = *a + *b + m[blake2b_sigma[r][2 * i + 1]];
+  *d = rotr64(*d ^ *a, 16);
+  *c = *c + *d;
+  *b = rotr64(*b ^ *c, 63);
```

Comment:
It would be nice if we could reuse the code from the macro, e.g. by just calling `G(r,i,*a,*b,*c,*d)` here and renaming the function to `g` so that the names don't clash. In that case `m` would need to be added as a parameter to the `G` macro. It would also be nice to do the same for the `ROUND` macro and function. Maybe introduces more complexity than I think. WDYT?

Reply:
Done in https://github.com/trezor/trezor-firmware/pull/4107/commits/07377d792d8692b8a4fd29c36fdf562310c34a76.